### PR TITLE
Telliot doc updates for the cli changes

### PR DESCRIPTION
**Pull request by lodainn**
- Fix --config flag usage that was causing error
- Add instructions for ProfitThreshold

### CODE REVIEW

**Comment by krasi-georgiev** [comment]
I am not sure why the linting fails, but if you run `make lint` locally should be able to see what changes does it apply.

**Comment by lodainn** [comment]
I think it's some sort of a tooling error rather than the code change itself:

image

**Comment by krasi-georgiev** [comment]
just run `make lint` locally and commit again

File: docs/configuration.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ Telliot commands and config file options are as the following:
 * `disputeTimeDelta` - how far back to store values for min/max range - default 5 \(in minutes\)
 * `disputeThreshold` - percentage of acceptable range outside min/max for dispute checking - default
 * `psrFolder` - folder location holding your psr.json file, default working directory
+* `ProfitThreshold` - Solutions will only be submitted when the estimated profit margin is greater than the `ProfitThreshold` setting.  The estimated profit margin (%) is defined as `100*(EstimatedRewards - EstimatedTransactionCosts)/EstimatedTransaction Costs`.  Setting a ProfitThreshold of 0 will submit solutions if the transaction is estimated to break even `(EstimatedRewards = EstimatedTransactionCosts)`.   Values are estimates only.  There is no guarantee on the estimated values or making a profit.
```
Comment (krasi-georgiev) [inline docs/configuration.md:55]:
```suggestion
* `ProfitThreshold` - Solutions will only be submitted when the estimated profit margin is greater than the `ProfitThreshold` setting.  The estimated profit margin (%) is defined as `100*(EstimatedRewards - EstimatedTransactionCosts)/EstimatedTransaction Costs`.  Setting a ProfitThreshold of 0 will submit solutions if the transaction is estimated to break even `(EstimatedRewards = EstimatedTransactionCosts)`.   Values are estimates only.  There is no guarantee on the estimated values or making a profit.
```

**Review by krasi-georgiev — APPROVED** [review]

**Comment by krasi-georgiev** [comment]
There was an extra space at the end of the sentence. I removed it and now the tests are passing.

**Pull request merged**